internal/keys/generate.go

```diff
@@ -30,8 +30,9 @@ import (
 )
 
 type flagsGenerate struct {
-	Seed       string `flag:"seed" info:"Deterministic seed phrase"`
-	KeySigAlgo string `default:"ECDSA_P256" flag:"sig-algo" info:"Signature algorithm"`
+	Seed           string `flag:"seed" info:"Deterministic seed phrase"`
+	FromPrivateKey string `flag:"from-private-key" info:"Use user private key instead of random"`
```

Comment:
It would be nice to also support `FromPublicKey` which would get private key from public key :D just kidding